h5p-bildetema/src/components/Bildetema/Bildetema.tsx

```diff
@@ -84,13 +91,13 @@ export const Bildetema: React.FC<BildetemaProps> = ({ defaultLanguages }) => {
     }
   });
 
-  useEffect(() => {
+  const routes = (): JSX.Element => {
```

Comment:
This could be a `useMemo` to avoid running it on every render (I'm not sure what really happens under the React hood)